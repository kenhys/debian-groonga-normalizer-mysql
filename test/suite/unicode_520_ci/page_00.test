register normalizers/mysql

normalize NormalizerMySQLUnicode520CI "aBc"
