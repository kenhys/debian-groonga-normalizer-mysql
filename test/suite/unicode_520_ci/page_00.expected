register normalizers/mysql
[[0,0.0,0.0],true]
normalize NormalizerMySQLUnicode520CI "aBc"
[[0,0.0,0.0],{"normalized":"ABC","types":[],"checks":[]}]
